apps/condo/pages/property/[id]/index.tsx

```diff
@@ -63,21 +72,69 @@ const PropertyInfoPanel: React.FC<IPropertyInfoPanelProps> = ({ title, message,
     </FocusContainer>
 )
 
+const PropertyListWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  row-gap: 40px;
```

Comment:
<img width="1312" alt="image" src="https://user-images.githubusercontent.com/50069872/211342114-c4bcb28f-3684-4f2b-8c5f-1e284999f632.png">

row-gap has relatively low support across browsers, so how about antd space with vertical direction and 40 gap?
as far as I know, it can replace it with margins, if gap is not supported...